### PR TITLE
Add Quefrency Rev. 5

### PR DESCRIPTION
**Pull request by nooges**
## Description

- Add Quefrency Rev. 5
- Make `default65macro` be the new `default` keymap
- Remove ISO layouts not supported by Rev. 4 & 5 PCBs

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/quefrency/keymaps/default/keymap.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include QMK_KEYBOARD_H
 
+extern keymap_config_t keymap_config;
```
Comment (drashna) [inline keyboards/keebio/quefrency/keymaps/default/keymap.c:3]:
```suggestion
```

File: keyboards/keebio/quefrency/keymaps/default60/keymap.c [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/keebio/quefrency/keymaps/default60/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

File: keyboards/keebio/quefrency/keymaps/default60/keymap.c [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
```
Comment (drashna) [inline keyboards/keebio/quefrency/keymaps/default60/keymap.c:9]:
```suggestion
enum custom_layer {
    _BASE,
    _FN1,
};
```

File: keyboards/keebio/quefrency/rev5/config.h [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+#define SOFT_SERIAL_PIN D0
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* ws2812 RGB LED */
+#define RGB_DI_PIN E6
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/keebio/quefrency/rev5/config.h:56]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by keyboard-magpie — APPROVED** [review]
In terms of getting to grips with approvals, this all LGTM

**Pull request merged**